src/Stepper/StepperButton.js

```diff
@@ -0,0 +1,49 @@
+// @flow srict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Icon from '../Icon';
+import Touchable from '../Button/Touchable';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type ButtonProps = {|
+  +icon: React.Element<typeof Icon>,
+  +touchable: boolean,
```

Comment:
Does `touchable` need to be mandatory prop? Can we have it optional with default value?

Reply:
Good point.